actioncable/app/javascript/action_cable/subscription.js

```diff
@@ -56,21 +56,22 @@
 // The "AppearanceChannel" name is automatically mapped between the client-side subscription creation and the server-side Ruby class name.
 // The AppearanceChannel#appear/away public methods are exposed automatically to client-side invocation through the perform method.
 
-const extend = function(object, properties) {
-  if (properties != null) {
-    for (let key in properties) {
-      const value = properties[key]
-      object[key] = value
-    }
-  }
-  return object
-}
-
 export default class Subscription {
   constructor(consumer, params = {}, mixin) {
     this.consumer = consumer
     this.identifier = JSON.stringify(params)
-    extend(this, mixin)
+    Subscription.extend(this, mixin)
```

Comment:
I think we can use [Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) here (I'm using it in [anycable-client](https://github.com/anycable/anycable-client/blob/master/packages/core/create-cable/index.js#L223), for example)